Tasks/Semin/task1.py

```diff
@@ -0,0 +1,6 @@
+deposit = int(input('enter the deposit amount at BYN: '))
+term = int(input('enter the term of the deposit in months: '))
+percent = int(input('enter annual percentage: '))
+n = percent / 100
+amount_on_account = deposit * ((1 + n / 12 )**term)
+print (int(amount_on_account))
```

Comment:
but where are kopeiki?

the result should be rounded to the second digit after a point, this is a logical mistake, -15 points